Source/WebInspectorUI/UserInterface/Views/LogContentView.css

```diff
@@ -63,6 +63,11 @@
     border: none;
 }
 
+.navigation-bar > .item.find-banner.console:has(+ .log-scope-bar.force-hidden + .divider.force-hidden + .message-channel-scope-bar.force-hidden),
+.navigation-bar > .item.find-banner.console:has(+ .log-scope-bar.force-hidden + .console-other-filters-button) {
+    padding-inline-end: 3px;
+}
```

Comment:
can you attach some screenshots of before and after for this?  im not sure i really understand why this is necessary (or if maybe there's a different approach that is less complex)